ndb_api/common_helpers.go

```diff
@@ -35,6 +37,21 @@ func GetDatabaseEngineName(dbType string) string {
 	}
 }
 
+func GetDatabaseTypeFromEngine(engine string) string {
+	switch engine {
+	case common.DATABASE_ENGINE_TYPE_POSTGRES:
+		return common.DATABASE_TYPE_POSTGRES
+	case common.DATABASE_ENGINE_TYPE_MYSQL:
+		return common.DATABASE_TYPE_MYSQL
+	case common.DATABASE_ENGINE_TYPE_MONGODB:
+		return common.DATABASE_TYPE_MONGODB
+	case common.DATABASE_ENGINE_TYPE_MSSQL:
+		return common.DATABASE_TYPE_MSSQL
+	default:
+		return ""
```

Comment:
what does an empty engine mean in the context of the caller? Should you be returning an error instead if the provided engine is not supported, etc?

Reply:
You are right, we should be returning errors instead of returning empty strings or -1s. Discussed with Manav, we will address these comments in next PR since they are non-blocking (for now will focus on release).